pkg/artifactory/resource_artifactory_replication_config.go

```diff
@@ -275,6 +275,10 @@ func resourceReplicationConfigExists(d *schema.ResourceData, m interface{}) (boo
 	replicationName := d.Id()
 	_, resp, err := c.V1.Artifacts.GetRepositoryReplicationConfig(context.Background(), replicationName)
 
+	if resp == nil {
```

Comment:
Check `err` first. If you check error first you probably never need to check if `resp` is `nil`.